remote/github/github.go

```diff
@@ -249,6 +256,21 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	return err
 }
 
+func deploymentStatus(client *github.Client, r *model.Repo, b *model.Build, link string) error {
+	// the deployment ID is only available in the the link to the build as the last element in the URL
+	parts := strings.Split(b.Link,"/")
+	id, _ := strconv.Atoi(parts[len(parts)-1])
+	status := getStatus(b.Status)
```

Comment:
the deployment event can be triggered by GitHub, however, a deployment event can also be triggered directly using the Drone API without involving the GitHub API.

when a deployment is triggered using the Drone API a deployment status should not be triggered. We can detect this by looking at the Link which won't match the expected pattern.